src/pytorch_lightning/trainer/trainer.py

```diff
@@ -649,7 +649,6 @@ def _call_and_handle_interrupt(self, trainer_fn: Callable, *args: Any, **kwargs:
                 return self.strategy.launcher.launch(trainer_fn, *args, trainer=self, **kwargs)
             else:
                 return trainer_fn(*args, **kwargs)
-        # TODO(awaelchli): Unify both exceptions below, where `KeyboardError` doesn't re-raise
```

Comment:
This was about doing the reconcilitation, teardown and state resetting moreso than unifying the excepts

Reply:
But will we do it? It's not trivial. I changed my opinion from the past and today prefer not to have my name in todos in the source code. It's a personal preference. 

I can leave the todo and just remove my name.

Reply:
I'd say yes.

Sure, feel free to use this PR to remove your name. There are several occurrences



```python
src/pytorch_lightning/trainer/connectors/accelerator_connector.py:346:        # TODO: @awaelchli improve the error messages below
src/pytorch_lightning/trainer/connectors/accelerator_connector.py:471:                # TODO: @awaelchli improve error message
src/pytorch_lightning/trainer/connectors/accelerator_connector.py:480:                # TODO: @awaelchli improve error message
src/pytorch_lightning/lite/wrappers.py:102:        # TODO (@awaelchli): let the precision plugin handle the conversion
src/pytorch_lightning/loops/epoch/training_epoch_loop.py:493:        # TODO(@awaelchli): let training/eval loop handle logic around limit_*_batches and val_check_batch
src/pytorch_lightning/loops/fit_loop.py:94:        # TODO(@awaelchli): This setter is required by debugging connector (fast dev run), should be avoided
src/pytorch_lightning/loops/fit_loop.py:105:        # TODO(@awaelchli): This setter is required by debugging connector (fast dev run), should be avoided
src/pytorch_lightning/loops/fit_loop.py:162:        # TODO(@awaelchli): Move track steps inside training loop and move part of these condition inside training loop
src/pytorch_lightning/strategies/strategy.py:240:        # TODO (@awaelchli): standardize this across all plugins in Lightning and Lite. Related refactor: #7324
src/pytorch_lightning/strategies/strategy.py:247:        # TODO (@awaelchli): standardize this across all plugins in Lightning and Lite. Related refactor: #7324
src/pytorch_lightning/strategies/strategy.py:252:        # TODO (@awaelchli): standardize this across all plugins in Lightning and Lite. Related refactor: #7324
```